kardinal-cli/cmd/root.go

```diff
@@ -424,9 +424,6 @@ func isPortOpenAndHTTP(localPortStr string) error {
 	}
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusOK {
```

Comment:
It doesn't matter the status code in the server response, we just need to know that a running server is responding on that port.